paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/viewmodel/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -193,4 +196,44 @@ class PaymentSheetPlaygroundViewModel(
                 inProgress.postValue(false)
             }
     }
+
+    suspend fun createPaymentIntent(
+        paymentMethodId: String?,
```

Comment:
We should add parameters for amount and currency, so that we can take the ones used in the activity.